frontend/packages/schema-editor/src/components/SchemaInspector/ItemPropertiesTab.test.tsx

```diff
@@ -40,7 +40,18 @@ describe('ItemPropertiesTab', () => {
     validateTestUiSchema(uiSchemaNodes);
     renderWithProviders({
       appContextProps: { schemaModel: SchemaModel.fromArray(uiSchemaNodes) },
-    })(<ItemPropertiesTab selectedItem={uiSchemaNodes[2]}/>);
+    })(<ItemPropertiesTab selectedItem={uiSchemaNodes[2]} />);
     expect(screen.getByText(textMock('combination_inline_object_disclaimer'))).toBeDefined();
   });
+
+  it('should render explanation message if no node is root pointer', () => {
```

Comment:
Slik jeg forstår koden er det vel heller dette som blir testet her?
```suggestion
  it('should render explanation message if the selected item is a root node', () => {
```